packages/seal/src/key-server.ts

```diff
@@ -6,13 +6,15 @@ import { bls12_381 } from '@noble/curves/bls12-381';
 import { KeyServerMove } from './bcs.js';
 import {
 	InvalidGetObjectError,
+	InvalidKeyServerError,
```

Comment:
InvalidKeyServerError is currently returned when the user does not set the key servers correctly, let's add a different error InvalidKeyServerVersionError